src/hidapi/libusb/hid.c

```diff
@@ -1368,6 +1428,11 @@ static int hidapi_initialize_device(hid_device *dev, const struct libusb_interfa
 		init_xboxone(dev->device_handle, desc.idVendor, desc.idProduct, conf_desc);
 	}
 
+	/* Initialize NSO GameCube controllers */
+	if ((desc.idVendor == 0x057e) && (desc.idProduct == 0x2073)) {
```

Comment:
Maybe change this to a function so it's easier to update when we add support for additional controllers?

Reply:
Moved this to `is_ns2`.